### ISSUE TITLE
(3.4.0) Added Task doesnt appear without manually syncing

### ISSUE THREAD

**Issue by chaosben** [Ericsson/exchangecalendar#365]
After adding a new task to a tasks-calendar, the statusbars says "1 added" but the it doesnt show up in the list. Only after syncing the calendars manually (or clearing the offline-cache via EWS-Settings->Maintenance) the new task appears.

Add-On-Version: 3.4.0
TB-Version: 38.3.0
Server-Version 15.0.775.35

Tanks for thinking about!

**Comment by bavincen** [comment]
Lightning version should be 4.0.0.1 or lesser

**Comment by chaosben** [comment]
thanks :)

**Comment by bavincen** [comment]
we are supporting 4.0.3 lightning
